tests/integration/test_fim/test_files/test_ambiguous_confs/test_ignore_works_over_restrict.py

```diff
@@ -164,35 +168,38 @@ def test_ignore_works_over_restrict(folder, filename, triggers_event, tags_to_ap
 
     tags:
         - scheduled
-        - time_travel
     '''
     logger.info('Applying the test configuration')
     check_apply_test(tags_to_apply, get_configuration['tags'])
-    scheduled = get_configuration['metadata']['fim_mode'] == 'scheduled'
 
     # Create file that must be ignored
     logger.info(f'Adding file {os.path.join(testdir1, filename)}, content: ""')
     create_file(REGULAR, folder, filename, content='')
 
-    # Go ahead in time to let syscheck perform a new scan if mode is scheduled
-    logger.info(f'Time travel: {scheduled}')
-    check_time_travel(scheduled, monitor=wazuh_log_monitor)
+    # Timeout for FIM to generate the log.
+    test_timeout = 5  # seconds
+    logger.info(f'Waiting up to {test_timeout} seconds for FIM to generate the log')
 
     if triggers_event:
         logger.info('Checking the event...')
-        event = wazuh_log_monitor.start(timeout=global_parameters.default_timeout,
-                                        callback=callback_detect_event,
-                                        error_message=f'Did not receive expected "Sending FIM event" '
-                                                      f'event for file {os.path.join(testdir1, filename)}').result()
+        event = wazuh_log_monitor.start(
+            timeout=test_timeout,
+            callback=callback_detect_event,
+            error_message=f'Did not receive expected "Sending FIM event" '
+                          f'event for file {os.path.join(testdir1, filename)}',
+            ).result()
```

Comment:
Why you change the previous lines (183-186)? Did it cause failures?

Reply:
No, it was because of the formatter I ran. It was restored as it was previously in commit [b6dcc84](https://github.com/wazuh/wazuh-qa/pull/3078/commits/b6dcc84036d83f6326e7f8757d7addb4e3c94de7)